security/pkg/nodeagent/plugin/providers/google/stsclient/stsclient.go

```diff
@@ -41,7 +41,7 @@ var (
 const (
 	httpTimeout = time.Second * 5
 	contentType = "application/json"
-	scope       = "https://www.googleapis.com/auth/cloud-platform"
+	Scope       = "https://www.googleapis.com/auth/cloud-platform"
```

Comment:
Change it to lower case to be consistent with the rest?

Reply:
This constant is also used in Scope:            stsclient.Scope,
https://github.com/istio/istio/blob/22e14891242e207a2922276bcec115b646c55e40/security/pkg/nodeagent/caclient/credentials.go#L151.

If it is lower case, then it will be not accessible at the above line.

Reply:
I see. Thanks.